Packs/CommonScripts/ReleaseNotes/1_15_56.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### StixCreator
+
+- Added new functionality to the `STIX2XSOARParser` object in the `TAXII2ApiModule` module.
```

Comment:
I suggest changing the unrelated packs RN to:
```suggestion
Fixed an issue where indicators without names were not properly parsed.
**Note:** This update was made to shared code used by both the integration and the ***Unit 42 ATOMs Feed*** integration. However, it does not affect this integration.
```

Reply:
done